homeassistant/components/history/__init__.py

```diff
@@ -142,7 +146,7 @@ async def ws_get_statistics_during_period(
     else:
         end_time = None
 
-    statistics = await hass.async_add_executor_job(
+    statistics = await get_instance(hass).async_add_executor_job(
```

Comment:
Would things break if this is accidentally called with `statistics = await hass.async_add_executor_job(`?

Reply:
It will work just fine.

For SQLite it will use NullPool as it did before but log a warning that it will be slower and to use the new method

Reply:
I changed it to use `report` 

Bonus find https://github.com/home-assistant/core/issues/68282

Reply:
```2022-03-16 18:01:08 WARNING (MainThread) [homeassistant.helpers.frame] Detected integration that Database access is slower in the default executor, use homeassistant.components.recorder.get_instance(hass).async_add_executor_job() for database operations. Please report issue for statistics using this method at homeassistant/components/statistics/sensor.py, line 509: if results := execute(query, to_native=False, validate_entity_ids=False):```

In this case its actually happening in the event loop 😢

Reply:
I update the message so it make sense if its called from the default executor or the event loop 😢